onnx-devel/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7-slim
+
```

Comment:
the folder name may be either "onnx-dev" or "onnx-develop", I think.

Reply:
Renamed. Thanks!